functional-tests/README.md

```diff
@@ -28,8 +30,12 @@ bitcoind --version
 
 ### Poetry
 
+> **_Note:_** Make sure you have installed Python 3.10 or higher.
```

Comment:
We can use github admonitions (https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

```suggestion
> [!NOTE]
> Make sure you have installed Python 3.10 or higher.
```